main.go

```diff
@@ -523,6 +523,56 @@ func main() {
 			return err
 		}
 
+		//
+		// hc-static-site
+		//
+		// There's enough different about this repo's config that it makes
+		// sense just to start from scratch.
+		hcStaticSiteRepositoryArgs := github.RepositoryArgs{
+			Name:                pulumi.String("hc-static-site"),
+			Description:         pulumi.String("Static website"),
+			Visibility:          pulumi.String("private"),
+			HasDownloads:        pulumi.Bool(true),
+			HasIssues:           pulumi.Bool(true),
+			HasProjects:         pulumi.Bool(true),
+			HasWiki:             pulumi.Bool(false),
+			VulnerabilityAlerts: pulumi.Bool(false),
+			AllowAutoMerge:      pulumi.Bool(false),
```

Comment:
I think this is useful to enable.

Reply:
For all of these, you have to turn these on for the import to work and then remove them again later. Pulumi will only import an existing resource if the state matches exactly